one-ui/ui/src/components/entities/entity-tiles.tsx

```diff
@@ -63,28 +75,47 @@ const EntityTiles: React.FC = () => {
             } 
           } catch (error) {
               let message = error.response.data.message;
-              console.log('Error while deleting load data artifact.', message);
+              console.log('Error while deleting mapping artifact.', message);
               setIsLoading(false);
-              //handleError(error);
           }
     }
 
     const createMappingArtifact = async (mapObj) => {
-        console.log('Create API Called!')
         try {
             setIsLoading(true);
       
             let response = await axios.post(`/api/artifacts/mapping/${mapObj.name}`, mapObj);
             if (response.status === 200) {
-              console.log('Create/Update LoadDataArtifact API Called successfully!')
+              console.log('Create MappingArtifact API Called successfully!')
               setIsLoading(false);
+              return true;
+            } else {
+                return false;
             }
           }
           catch (error) {
-            let message = error.response.data.message;
-            console.log('Error While creating the Load Data artifact!', message)
+            let message = error;
+            console.log('Error While creating the Mapping artifact!', message)
             setIsLoading(false);
-            //handleError(error);
+            return false;
+          }
+    }
+
+    const updateMappingArtifact = async (mapObj) => {
```

Comment:
Can this be "mapping" instead of "mapObj"? "mapObj" makes me think it's different from a "mapping" in terms of what data it holds. If they're the same, let's only use one term, that's simpler.